### ISSUE TITLE
Terminal stuck in "Connecting..." state

### ISSUE THREAD

**Issue by glatino-pdx** [kontena/lens#5835]
**Describe the bug**
When you try to open the shell of any pod, the terminal window remains permanently in "Connecting", with the warning of the attached image

**To Reproduce**
1. Connect to a cluster
2. Click on a pod
3. Click on the 3 dots and choose "Shell"
4. The bottom terminal remain stuck on Connecting state

**Expected behavior**
The terminal should be working; the issue is not present on stable version.

**Screenshots**
![Screenshot 2022-07-14 115819](https://user-images.githubusercontent.com/105722462/178957709-90d78abe-0f86-42e9-ab86-6ac2fd6c6f2b.png)
![Screenshot 2022-07-14 120125](https://user-images.githubusercontent.com/105722462/178957729-d37d88e8-9c25-4b9d-a9c1-b9c692f4894a.png)

**Environment (please complete the following information):**
- Lens Version: 5.6.0-alpha.7.20220713.1
- OS: Win10 Pro x64 21H2
- Installation method: installer

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
no output logs from CLI
no error lines in C:\Users\xxx\AppData\Roaming\Lens\logs

**Kubeconfig:**
not needed

**Additional context**
Powershell v5.1.19041.1682

**Comment by jim-docker** [comment]
Seeing this in alpha releases:
```
info:    ▪ [SHELL-SESSION]: PTY for 8b4a49225587e03274806a10e60ec305:3ed17d5e-23ba-46a0-a5ce-72912dbca25e is started with PID=13932 +237ms
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
Error: HTTP Error (403)
    at ClientRequest.<anonymous> (C:\Program Files\Lens\resources\app.asar\node_modules\@sentry\electron\main\transports\electron-net.js:139:36)
    at ClientRequest.emit (node:events:394:28)
    at ClientRequest.emit (node:domain:470:12)
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:105:6816)
    at SimpleURLLoaderWrapper.emit (node:events:394:28)
    at SimpleURLLoaderWrapper.emit (node:domain:470:12)
(node:9744) UnhandledPromiseRejectionWarning: Error: HTTP Error (403)
    at ClientRequest.<anonymous> (C:\Program Files\Lens\resources\app.asar\node_modules\@sentry\electron\main\transports\electron-net.js:139:36)
    at ClientRequest.emit (node:events:394:28)
    at ClientRequest.emit (node:domain:470:12)
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:105:6816)
    at SimpleURLLoaderWrapper.emit (node:events:394:28)
    at SimpleURLLoaderWrapper.emit (node:domain:470:12)
(node:9744) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 8)
```
but can't reproduce in dev version.

**Comment by jim-docker** [comment]
Terminal works ok when running `make dev`
Terminal hangs when running app installed by installer output by `make build`. Javascript error is shown in dialog:

![Screen Shot 2022-07-26 at 9 42 53 PM](https://user-images.githubusercontent.com/40840436/181142479-9d50f6b1-f6e2-4880-b897-006b7dfe2ca9.png)

something goes wrong in electron-builder, which rebuilds the native module node-pty and at run-time the module (or part of it) can't be found.

I tried using node-pty versions 0.11.0-beta9, -beta11, and beta19 with same result. I also tried newer version of electron-builder (23.3.2)

In the end I tried the previously used node-pty stable version of 0.10.1 and it fixes this issue, and also appears to work fine on macos and linux, despite comment here: https://github.com/lensapp/lens/pull/5562#issue-1261575055. See #5916
